core/src/test/java/org/bouncycastle/crypto/test/AESWrapPadTest.java

```diff
@@ -162,6 +163,58 @@ public void performTest()
             rnd.nextBytes(keyToWrap);
             wrapAndUnwrap(kek, keyToWrap);
         }
+
```

Comment:
All the existing tests checked correct behaviour. I've added tests for incorrect behaviour.